### PR TITLE
Feature/#26 Create error page

### PR DESCRIPTION
**Pull request by MangriMen**

### CODE REVIEW

**Comment by MangriMen** [comment]
[Create error page](https://trello.com/c/RX4swUlI/26-create-error-page)

**Comment by FFreestyler** [comment]
image
I look at these numbers, and for some reason i don't like them. Maybe we can print without numbers?

**Review by FFreestyler — APPROVED** [review]

**Comment by MangriMen** [comment]
> image I look at these numbers, and for some reason i don't like them. Maybe we can print without numbers?

Font should be changed later

File: src/components/common/NotFoundPageComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+  return (
+    <ParBox
+      position="fixed"
+      height="100%"
+      width="100%"
+      // TODO Fix when bug/#30 is merged
+      sx={{
+        border: '0',
+      }}
```
Comment (StepanovRoma) [inline src/components/common/NotFoundPageComponent.tsx]: 30 bug was fixed, what should be there?

Comment (MangriMen) [reply src/components/common/NotFoundPageComponent.tsx]: Fixed

**Review by StepanovRoma — APPROVED** [review]

**Pull request merged**